### PR TITLE
New doc

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -84,13 +80,126 @@ Annotation processing has quite a few advantages over external tools or build to
...
+
+## Hello Quarkus
+
+Add the following dependency to your project:
+
+```xml
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-kubernetes</artifactId>
+  <version>1.0.0/version>
```
Comment (geoand) [inline readme.md]: Should be `1.0.0.Final`

File: readme.md [hunk 2]
```diff
@@ -248,6 +371,11 @@ public class Main {
 
 Additional options are provided for adding environment variables from fields, config maps and secrets. 
 
+To add an additonal environment variables using `application.properties`:
```
Comment (geoand) [inline readme.md]: `additonal` -> `additional`

File: readme.md (hunk 1 above)
Comment (iocanel) [reply readme.md]: Fixed

File: readme.md (hunk 2 above)
Comment (iocanel) [reply readme.md]: Fixed

**Pull request merged**